app/src/main/java/com/cornellappdev/android/eateryblue/ui/navigation/MainTabbedNavigation.kt

```diff
@@ -278,16 +274,17 @@ fun SetupNavHost(
                     Routes.LEGAL to { navController.navigate(Routes.LEGAL.route) },
                     Routes.PRIVACY to { navController.navigate(Routes.PRIVACY.route) },
                     Routes.SUPPORT to { navController.navigate(Routes.SUPPORT.route) },
-//                    Routes.PROFILE to {
-//                        navController.navigate("${Routes.PROFILE.route}/false") {
-//                            if (navController.isOnBackStack(Routes.ACCOUNT.route)) {
-//                                popUpTo(Routes.ACCOUNT.route) { inclusive = true }
-//                            } else {
-//                                popUpTo(Routes.SETTINGS.route) { inclusive = true }
-//                            }
-//                        }
-//                    }
-                )
+                    Routes.PROFILE to {
+                        navController.navigate("${Routes.PROFILE.route}/false") {
+                            if (navController.isOnBackStack(Routes.ACCOUNT.route)) {
+                                popUpTo(Routes.ACCOUNT.route) { inclusive = true }
+                            } else {
+                                popUpTo(Routes.SETTINGS.route) { inclusive = true }
+                            }
+                        }
+                    }
```

Comment:
Try to reverse this logic to make it go to the settings screen if the settings screen is on the back stack. This logic is like reversed and makes no sense cuz the settings screen must be navigated to from the account screen lol